package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "20.6.2",
+  "version": "20.6.4",
```

Comment:
version bumped twice 🤷🏽‍♂️

Reply:
I also bumped the version in https://github.com/department-of-veterans-affairs/vets-json-schema/pull/612